BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/coordinates/CoordinateState.java

```diff
@@ -0,0 +1,7 @@
+package pl.jakubowskiprzemyslaw.tajgertim.models.coordinates;
+
+public enum CoordinateState {
```

Comment:
Maybe something like FieldState to contain state. And Field is connected with Coordinate in some way

Reply:
Changed